cpp/src/db/db_object.cu

```diff
@@ -304,9 +277,8 @@ void db_result<idx_t>::allocateColumns(idx_t size) {
   if (dataValid)
     throw new std::invalid_argument("Already allocated columns");
```

Comment:
We should use CUGRAPH_EXPECTS and CUGRAPH_FAIL for consistency.

Reply:
Changed to use CUGRAPH_EXPECTS rather than just throwing an exception.